### PR TITLE
Fix aarch64 support

### PR DESCRIPTION
**Pull request by yeongrokgim**
## Fixes

- Improve README.md: readability, informational hierarchy, applying lint
- Refactor `python/configure.sh` to properly support more platforms (`Linux`/`Darwin`, `x86_64`/`aarch64`(`arm64`))
- Specify numpy version to avoid build numpy from source (list of available version/platform obtained from http://pypi.org/pypi/numpy/json)
- Specify minimum pip version that supports install numpy from whl(pre-built binary) rather than build from source

## Details

- ubuntu18(jetpack4.6) has `pip==9.0.1` by default
  - installed from `apt-get install python3-pip`
  - fails to install numpy from pre-built package
- ubuntu20(jetpack5.0) has `pip==20.0.2`
- pre-built binary for numpy is available for:
  - `linux/aarch64` starting from `1.19.0`
  - `macos/arm64` staring from `1.21.0`

### CODE REVIEW

File: build_sdk.sh [hunk 1]
```diff
@@ -23,7 +22,7 @@ function make_python_sdk() {
 if [ $# -ne 1 ]; then
     echo "Usage: $0 generating-language"
     echo "Supported languages: cpp, javascript, python"
-    exit -1
+    exit 1
```
Comment (thor-sr) [inline build_sdk.sh:25]: Just curious, what's the reason behind this change?

File: python/configure.sh [hunk 2]
```diff
@@ -1,30 +1,123 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (thor-sr) [inline python/configure.sh:1]: Good to know :+1:

**Review by thor-sr — APPROVED** [review]
:man_scientist:

File: build_sdk.sh (hunk 1 above)
Comment (yeongrokgim) [reply build_sdk.sh:25]: > Note in particular that `exit -1` is equivalent to `exit 255`, but that `exit 1` is the more canonical way of expressing the first possible error code.
- https://www.shellcheck.net/wiki/SC2242
- https://tldp.org/LDP/abs/html/exitcodes.html

**Review by sebastian-seoul-robotics — APPROVED** [review]
